deploy/rbd/kubernetes/v1.14+/helm/ceph-csi-rbd/templates/nodeplugin-daemonset.yaml

```diff
@@ -107,6 +107,28 @@ spec:
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
           resources:
+{{ toYaml .Values.nodeplugin.plugin.resources | indent 12 }}
+        - name: liveness-prometheus
+          image: "{{ .Values.nodeplugin.plugin.image.repository }}:{{ .Values.nodeplugin.plugin.image.tag }}"
+          args:
+            - "--type=liveness"
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--livenessport=8081"
```

Comment:
@Madhu-1  What about making this `port` a configurable value . May be in another PR?

Reply:
yes, this can be added to Value.yaml @wilmardo isn't it?

 also, we can add all ceph-csi and sidecar configuration parameters to value.yaml  @wilmardo sounds like a good idea?

Reply:
Yeah had this as an improvement but decided to keep this PR simple, just to sync 1.14+ with the 1.13 charts.

Will follow up this PR with those values split into the values.yaml of all the charts :)

Reply:
@wilmardo Thanks :)